recipes/plannotate/meta.yaml

```diff
@@ -1,48 +1,46 @@
-{% set version = "1.2.0" %}
+{% set version = "1.2.1" %}
 
 package:
   name: plannotate
   version: {{ version }}
 
 build:
-  number: 4
+  number: 1
```

Comment:
should be reset to 0

Reply:
Sorry, thank you -- I released I forgot to change the sha256  and I am pointing to the wrong loc for the db download as well